### PR TITLE
[show] session_id

### PR DESCRIPTION
**Pull request by rtxm**

### CODE REVIEW

File: examples/recognition/basic_async.py [hunk 1]
```diff
@@ -78,7 +78,13 @@ async def expect(*event_classes):
                 )
             )
 
-            await send(client.set_params(speech_language="fr", no_input_timeout=5000))
+            await send(
+                client.set_params(
+                    speech_language="fr",
+                    no_input_timeout=5000,
+                    logging_tag="basic_async_test",
```
Comment (merwan) [inline examples/recognition/basic_async.py:85]: What is this `logging_tag`?

**Review by merwan — APPROVED** [review]
LGTM

File: examples/recognition/basic_async.py (hunk 1 above)
Comment (rtxm) [reply examples/recognition/basic_async.py:85]: https://app.shortcut.com/allomedia/story/46544/gather-the-logging-tag-header-and-add-it-to-the-metric-payload

Comment (rtxm) [reply examples/recognition/basic_async.py:85]: The difference between `session-id` and `logging-tag` is that the former is for the complete session and the latter is used to tag a contiguous sequence of interactions within the session.

Comment (merwan) [reply examples/recognition/basic_async.py:85]: Thanks for the explanations

**Pull request merged**